IAMOTZ/ievents
# #152986588 Fix template bugs

#### What does this PR do?
Fix the bugs that are on the template pages
#### Description of Task to be completed?
Implement feedback on UI template
#### How should this be manually tested?
- clone the repo
- cd into the template directory
- open the index.html with any browser of your choice
- the template should open in that browser
- navigate through the pages to see the changes 
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152986588
#### Screenshots (if appropriate)
None
#### Questions:
None
